backend-shared/exportfuncs.cpp

```diff
@@ -34,32 +34,27 @@ bool ExportCallback::canceled() const
 
 #if !defined(SUBSURFACE_MOBILE)
 
-#include "desktop-widgets/mainwindow.h" // Currently needed for profile printing. TODO: remove.
+// Let's say that 800x600 is a "reasonable" profile size. Use four times that for printing.
+static const constexpr int profileScale = 4;
+static const constexpr int profileWidth = 800 * profileScale;
+static const constexpr int profileHeight = 600 * profileScale;
```

Comment:
I thought that `constexpr` implied `const`. Why do you need both here? Or asked another way, what's the difference between
`static constexpr int ...`
and
`static const constexpr int ...`

Reply:
Yes `const` is weaker and redundant in this case. Removed.